onnx/defs/tensor/old.cc

```diff
@@ -4128,17 +4135,24 @@ ONNX_OPERATOR_SET_SCHEMA(
             return;
           }
 
-          std::vector<int64_t> axes;
-          if (!getRepeatedAttribute(ctx, "axes", axes)) {
-            return;
-          }
-
           if (!ctx.getInputType(0)->tensor_type().has_shape()) {
             return;
           }
 
           ctx.getOutputType(0)->mutable_tensor_type()->mutable_shape();
           const auto& input_shape = ctx.getInputType(0)->tensor_type().shape();
+          const auto input_ndim = input_shape.dim_size();
```

Comment:
That seems to be the same code than above, could it be moved in one separate function?